go get fails in Debian Jessie: /usr/bin/ld: cannot find -lGl

Prior to this, I fixed an include file `GL/glx.h` not found by `apt-get` installing `mesa-common-dev`.

```
$ go get github.com/go-gl/gl/v{3.2,3.3,4.1,4.4,4.5}-{core,compatibility}/gl
# github.com/go-gl/gl/v3.2-core/gl
/usr/bin/ld: cannot find -lGL
/usr/bin/ld: cannot find -lGL
collect2: error: ld returned 1 exit status
# github.com/go-gl/gl/v3.3-core/gl
/usr/bin/ld: cannot find -lGL
/usr/bin/ld: cannot find -lGL
collect2: error: ld returned 1 exit status
# github.com/go-gl/gl/v4.1-core/gl
/usr/bin/ld: cannot find -lGL
/usr/bin/ld: cannot find -lGL
collect2: error: ld returned 1 exit status
# github.com/go-gl/gl/v3.3-compatibility/gl
/usr/bin/ld: cannot find -lGL
/usr/bin/ld: cannot find -lGL
collect2: error: ld returned 1 exit status
# github.com/go-gl/gl/v3.2-compatibility/gl
/usr/bin/ld: cannot find -lGL
/usr/bin/ld: cannot find -lGL
collect2: error: ld returned 1 exit status
# github.com/go-gl/gl/v4.4-core/gl
/usr/bin/ld: cannot find -lGL
/usr/bin/ld: cannot find -lGL
collect2: error: ld returned 1 exit status
# github.com/go-gl/gl/v4.1-compatibility/gl
/usr/bin/ld: cannot find -lGL
/usr/bin/ld: cannot find -lGL
collect2: error: ld returned 1 exit status
# github.com/go-gl/gl/v4.5-core/gl
/usr/bin/ld: cannot find -lGL
/usr/bin/ld: cannot find -lGL
collect2: error: ld returned 1 exit status
# github.com/go-gl/gl/v4.4-compatibility/gl
/usr/bin/ld: cannot find -lGL
/usr/bin/ld: cannot find -lGL
collect2: error: ld returned 1 exit status
# github.com/go-gl/gl/v4.5-compatibility/gl
/usr/bin/ld: cannot find -lGL
/usr/bin/ld: cannot find -lGL
collect2: error: ld returned 1 exit status
```

--- a/README.md
+++ b/README.md
@@ -9,7 +9,8 @@ Features:
 - Support for extensions (including debug callbacks).
 
 Requirements:
-- A cgo compiler (typically gcc)
+- A cgo compiler (typically gcc).
+- The `libgl1-mesa-dev` package.
 
 Usage
 -----

--- a/README.md
+++ b/README.md
@@ -10,7 +10,7 @@ Features:
 
 Requirements:
 - A cgo compiler (typically gcc).
-- The `libgl1-mesa-dev` package.
+- On Ubuntu/Debian-based systems, the `libgl1-mesa-dev` package.
 
 Usage
 -----

